views/partials/_example.njk

```diff
@@ -35,6 +35,9 @@
     ```js
     {{ getNunjucksCode(examplePath) | safe }}
     ```
+    <p class="govuk-!-mt-r6 govuk-!-ml-r6">
```

Comment:
can we use `govuk-!-mt-r4` `govuk-!-ml-r4` on these so the spacing is equal?

(not for this PR but if this is to be common place then we should consider moving the close button to fit on the same baseline)